configs/raspberrypi_config

```diff
@@ -95,4 +95,34 @@ function platform_modify() {
     run_and_log_cmd "sudo unlink work/rootfs/etc/rc3.d/S01resize2fs_once"
   fi
   run_and_log_cmd "sudo rm -f work/rootfs/etc/init.d/resize2fs_once"
+
```

Comment:
Review note:

Added to rasperrypi_config in general. Is this appropriate?

Reply:
Good question... I think it's not very future proof. We expect that already in the next version of Raspbian, this will be solved, since the Systemd version should be high enough to take care of this. How about adding this to its own config file instead, and using that in the image that we produce for our downloads? The most important is that that image expands the data partition.

Reply:
I also think it would be best with a separate config file.

Reply:
Alright, goodie. I figured as much.

Next question. Should the config file then install this as it is sourced? Does anyone see any issues with this? 

Reason I'm asking, is it does not seem to me that adding multiple functions to the current hooks is possible? 

Meaning, that: In the new config file, this cannot be added to `platform_modify`, as this will override the hook added in `raspberrypi_config`? 

At least it seems to me to be this way. If yous agree, then my suggestion is to rewrite this into proper hooks, which one can add to?

Reply:
Suggestion: Where it calls `platform_modify` today, instead change it into this:

```bash
for platform_modifier in $PLATFORM_MODIFY_HOOKS; do
    $platform_modifier
done
```

And then make sure `PLATFORM_MODIFY_HOOKS=platform_modify` is present in the base config file. Then you can add function names to this variable to call additional hooks.

Reply:
Yes, I've been playing around with this actually. But we'd have to change all the hook functions. If you are okay with this, I will go ahead and do it.

@mirzak any reservations?

Reply:
Hooks PR: https://github.com/mendersoftware/mender-convert/pull/176

If this is okayed, I will go ahead and rebase this PR on top of it.